packages/cli/test/integration/users.api.test.ts

```diff
@@ -364,6 +364,8 @@ describe('PATCH /users/:id/role', () => {
 		NO_OWNER_ON_OWNER,
 	} = UsersController.ERROR_MESSAGES.CHANGE_ROLE;
 
+	const UNAUTHORIZED = 'Unauthorized';
```

Comment:
Protecting the endpoint with `user:changeRole` has made this check in `PATCH /users/:id/role` unreachable:

```ts
if (req.user.globalRole.scope === 'global' && req.user.globalRole.name === 'member') {
	throw new UnauthorizedError(NO_MEMBER);
}
```

Can we please remove the check and the error message and any references to it?